src/NuGet.Core/NuGet.Frameworks/FrameworkNameProvider.cs

```diff
@@ -194,6 +207,7 @@ public string GetVersionString(string framework, Version version)
                     versionParts.Pop();
                 }
 
+                // TODO: This comment needs to be updated (maybe the code, too)
```

Comment:
don't forget this todo.
yes. sorry, i removed two of those cases and didn't fix comment.